src/Apps/Order/Routes/Shipping2/Components/ShippingQuotes2.tsx

```diff
@@ -57,42 +65,63 @@ export const ShippingQuotes2: React.FC<ShippingQuotesProps> = ({
   }
 
   return (
-    <RadioGroup
-      onSelect={handleShippingQuoteSelected}
-      defaultValue={shippingContext.state.selectedShippingQuoteID}
-    >
-      {quotes.map(shippingQuote => {
-        const description =
-          shippingQuoteDescriptions[shippingQuote?.typeName as string]
-        const displayName =
-          shippingQuoteDisplayNames[shippingQuote?.typeName as string]
-
-        return (
-          <BorderedRadio
-            data-testid="shipping-quotes"
-            value={shippingQuote?.id}
-            key={shippingQuote?.id}
-            position="relative"
-          >
-            <Flex flexDirection="column" width="100%">
-              <GridColumns>
-                <Column span={10}>
-                  <Text variant="sm-display" textTransform="capitalize">
-                    {displayName}
-                  </Text>
-                  <Text textColor="black60">{description}</Text>
-                </Column>
-                <Column span={2} textAlign={"right"}>
-                  <Text textTransform="capitalize" data-testid="quotePrice">
-                    {shippingQuote?.price}
-                  </Text>
-                </Column>
-              </GridColumns>
-            </Flex>
-          </BorderedRadio>
-        )
-      })}
-    </RadioGroup>
+    <CollapseDetails>
+      <Text variant="sm">Artsy shipping options</Text>
+
+      <Text variant="xs" mb="1" color="black60">
+        {orderData.isOffer ? (
+          <>
+            Please note that these are estimates and may change once offer is
+            finalized. All options are eligible for Artsy’s Buyer Protection
+            policy, which protects against damage and loss.
+          </>
+        ) : (
+          <>
+            All options are eligible for Artsy’s Buyer Protection policy, which
+            protects against damage and loss.
+          </>
+        )}
+      </Text>
+
+      <RadioGroup
```

Comment:
This previously was in the index.tsx file - all shipping quotes stuff now in this component